hackathon_site/hackathon_site/urls.py

```diff
@@ -18,19 +18,16 @@
 from django.conf.urls import url
 from django.conf import settings
 
-from rest_framework import permissions
 from drf_yasg.views import get_schema_view
 from drf_yasg import openapi
 
-PUBLIC = bool(settings.DEBUG)
-
 schema_view = get_schema_view(
     openapi.Info(
         title="Hardware Hackathon API",
         default_version="v1",
         description="API Endpoint Visualization for Hardware Hackathon",
     ),
-    public=PUBLIC,
+    public=bool(settings.DEBUG),
```

Comment:
In case anyone is curious about this, I moved this inline to reduce confusion beyond this. On my first attempt, I figured that the `PUBLIC` variable was `not settings.DEBUG` (without actually reading it), so I tried to setup the debug toolbar with

```python
if not PUBLIC:
  ...
```

which follows semantically imo.